docs: add info about declaration (#10)
patak-dev/vite-plugin-virtual: README.md

invalidateVirtualModule(server, 'virtual:config')
```

> ## Types 
> 
> To enable TypeScript IntelliSense for a virtual module, you need to declare the module:

Reviewer: ```suggestion
## Types

To enable TypeScript IntelliSense for a virtual module, you can use ambient modules to declare the module in a `.d.ts` file included by your TS setup:
```

```ts

> // vite-env.d.ts

Reviewer: ```suggestion
```

declare module 'virtual:config' {
  export const hello: string;
}
```
   

> For more details about declaration, you can refer to the official Vite documentation on [IntelliSense for TypeScript](https://vitejs.dev/guide/env-and-mode#intellisense-for-typescript).

Reviewer: ```suggestion
For more details refer to the [TypeScript docs for ambient modules](https://www.typescriptlang.org/docs/handbook/modules/reference.html#ambient-modules).
```

> ## Types 
> 
> To enable TypeScript IntelliSense for a virtual module, you need to declare the module:
> 
> ```ts
> // vite-env.d.ts
> declare module 'virtual:config' {
>   export const hello: string;
> }
> ```
>    
> For more details about declaration, you can refer to the official Vite documentation on [IntelliSense for TypeScript](https://vitejs.dev/guide/env-and-mode#intellisense-for-typescript).
> 

Reviewer: ```suggestion
## Types

To enable TypeScript IntelliSense for a virtual module, you can use ambient modules to declare the module in a `.d.ts` file included by your TS setup:

```ts
declare module 'virtual:config' {
  export const hello: string;
}
```

For more details refer to the [TypeScript docs for ambient modules](https://www.typescriptlang.org/docs/handbook/modules/reference.html#ambient-modules).

```
